config/profiler.php

```diff
@@ -14,7 +14,7 @@
 $config['memory_usage']         = TRUE;
 $config['post']                 = TRUE;
 $config['queries']              = TRUE;
-$config['eloquent']             = FALSE;
+$config['eloquent']             = TRUE;
```

Comment:
but those effect when common people using composer. the config active as git repo in thirdparty. so what the best solution to trigger TRUE and FALSE for enable eloquent ?